arches/install/arches-templates/package.json

```diff
@@ -5,14 +5,15 @@
         "build_development": "yarn eslint:check && yarn ts:check && cross-env NODE_OPTIONS=--max-old-space-size=2048 webpack --config ./webpack/webpack.config.dev.js",
         "build_production": "yarn eslint:check && yarn ts:check && cross-env NODE_OPTIONS=--max-old-space-size=2048 NODE_ENV=production webpack --config ./webpack/webpack.config.prod.js",
         "build_test": "yarn eslint:check && yarn ts:check && cross-env NODE_OPTIONS=--max-old-space-size=2048 webpack --config ./webpack/webpack.config.dev.js --env test=true",
-        "eslint:check": "eslint . --ext .vue,.ts",
+        "eslint:check": "eslint **/src",
+        "eslint:fix": "eslint **/src --fix",
         "eslint:watch": "nodemon --watch . --ext ts,vue --exec yarn --silent eslint:check",
```

Comment:
Do we need to rip out `--ext` here, too?

Reply:
nope, this `--ext` is for nodemon